docs/plugin-proposal-decorators.md

```diff
@@ -114,15 +130,13 @@ Use the legacy (stage 1) decorators syntax and behavior.
 
 If you are including your plugins manually and using `@babel/plugin-proposal-class-properties`, make sure that `@babel/plugin-proposal-decorators` comes _before_ `@babel/plugin-proposal-class-properties`.
 
-When using the `legacy: true` mode, the [`setPublicClassFields` assumption](assumptions.md#setpublicclassfields) must be enabled to support the `@babel/plugin-proposal-decorators`.
```

Comment:
This has not been necessary for years - https://github.com/babel/babel/discussions/14133